core/domain/services/graphql/data/mutations/DatetimeMutation.php

```diff
@@ -123,4 +128,21 @@ public static function setRelatedTickets(EE_Datetime $entity, array $tickets)
             }
         }
     }
+
+
+    /**
+     * Sets the venue for the datetime.
+     *
+     * @param EE_Datetime $entity The datetime instance.
+     * @param int      $venue  The venue ID
+     * @throws EE_Error
+     * @throws InvalidDataTypeException
+     * @throws InvalidInterfaceException
+     * @throws InvalidArgumentException
+     * @throws ReflectionException
+     */
+    public static function setVenue(EE_Datetime $entity, int $venue)
+    {
+        $entity->add_venue($venue);
+    }
```

Comment:
If an empty value is passed for `$venue`, it means that the user unselected the venue for the datetime. Shouldn't we remove the venue in that case?